.github/workflows/update-bridge.yml

```diff
@@ -1,91 +0,0 @@
-# WARNING: This file is autogenerated - changes will be overwritten if not made via https://github.com/pulumi/ci-mgmt
```

Comment:
Is this file deletion intended? Seems out of scope for the purpose of updating Java.

Reply:
It is. It was caused by running `make ci-mgmt`, since deleting that file is part of a ci-mgmt change.